### DISCUSSION TITLE
How to setup Azure Blob Storage with Hierarchical Namespace?

### DISCUSSION THREAD

**Discussion by AndiRudi** [microsoft/aspire#5715] [Q&A] [answered]
We wanted to use directories in Azure Blob Storage and after a long confusion we realised that this is only supported in Gen2/Azure Data Lake. The publishing of Azure Blob Storage per default does not enabled Hierarchical Namespaces so I was wondering how to set this up?

**Comment by davidfowl** [comment, marked as answer]
You’ll be able to configure this in code using the azure cdk, but it’s not ready until Aspire 9
